plasmapy/atomic/tests/test_particle_class.py

```diff
@@ -192,6 +200,7 @@
       'element_name': 'hydrogen',
```

Comment:
@namurphy Why is deuterium listed as hydrogen? Is this for convenience? Should we add an isotope name property and spell out deuterium, tritium, etc?

Reply:
sounds good